@utahdts/utah-design-system/react/contexts/UtahDesignSystemContext/hooks/useBanner.js

```diff
@@ -0,0 +1,66 @@
+// @ts-check
+import { useCallback, useMemo } from 'react';
+import useUtahDesignSystemContext from '../useUtahDesignSystemContext';
+import uuidv4 from '../../../util/uuidv4';
+
+/** @typedef {import('../../../jsDocTypes').UtahDesignSystemContextBanner} UtahDesignSystemContextBanner */
+/** @typedef {import('../../../jsDocTypes').BannerPlacement} BannerPlacement */
+/**
+ * @returns {{
+ * addBanner: function({
+ *  className: string | undefined,
+ *  duration: number | undefined,
+ *  icon: HTMLElement | undefined,
+ *  message: HTMLElement | string,
+ *  position: BannerPlacement | undefined
+ * }): void,
+ * removeBanner: function(UtahDesignSystemContextBanner): void
+ * }}
+ */
+export function useBanner() {
+  const [, setState] = useUtahDesignSystemContext();
+
+  // eslint-disable-next-line function-paren-newline
+  const addBanner = useCallback(
+    /**
+     * @param {string | undefined} [className]
```

Comment:
Change to:
* @param {string} [className]

Don't use "| undefined" unless you really want them to use "undefined"